ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -327,4 +354,45 @@ private static boolean bug1733118Affected(String namespace, String docType, Obje
             .matches("0[.]([0-9]|1[0-6])[.].*"); // < 0.17
   }
 
+  // See bug 1751753 for explanation of context.
+  private static void processForBug1751753(ObjectNode json) {
+    // Sanitize keys in the SEARCH_COUNTS histogram.
+    JsonNode searchCounts = json.path("payload").path("keyedHistograms").path("SEARCH_COUNTS");
+    processKeysForBug1751753(searchCounts);
```

Comment:
IMO it would be cleaner to check if this was an object before calling `processKeysForBug1751753()`, and then you could cast it as `ObjectNode` on the way in and avoid doing that check/cast in `processKeysForBug1751753()`.
```suggestion
    if (searchCounts.isObject()) {
      processKeysForBug1751753((ObjectNode) searchCounts);
    }
```